### PR TITLE
Quiz Meets Rewrite

### PR DESCRIPTION
**Pull request by scottpeterson**

### CODE REVIEW

File: content/quiz_meets.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Quiz Meets
+The tournament brackets are based on three things:   A preliminary round, an elimination round (at the discretion of the meet director), and championship quizzes.
 
 ## Preliminary Rounds
+The teams will all receive team points as follows:
+- First place in a quiz:  10 points, plus 1 point for every 10 points over 100
+- Second place in a quiz:  5 points, plus 1 point for every 10 points over 60
+- Third place in a quiz:  1 point, plus 1 point for every 10 points over 30
+In case of a tie, points are awarded according to the team score at the end of question 20.
+If ties are not being broken in prelims, more than one team can receive First place, Second place, or Third place points.
```
Comment (scottpeterson) [inline content/quiz_meets.md:10]: Added by me.

File: content/quiz_meets.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 # Quiz Meets
+The tournament brackets are based on three things:   A preliminary round, an elimination round (at the discretion of the meet director), and championship quizzes.
 
 ## Preliminary Rounds
+The teams will all receive team points as follows:
+- First place in a quiz:  10 points, plus 1 point for every 10 points over 100
```
Comment (scottpeterson) [inline content/quiz_meets.md:6]: I decided not to define "Raw Points" and "Calculated Team Points"

File: content/quiz_meets.md [hunk 3]
```diff
@@ -25,5 +32,111 @@
     This is the end of the block.
 
 ## Elimination Rounds
```
Comment (scottpeterson) [inline content/quiz_meets.md:34]: I kept all of this verbatim. Don't feel super strongly.

File: content/quiz_meets.md [hunk 4]
```diff
@@ -25,5 +32,111 @@
...
+Quiz J:    First Quiz D, First Quiz G, First Quiz I
+Quiz K:        Second Quiz J, First Quiz J, Third Quiz J
+Quiz L:       Second Quiz K, First Quiz K, Third Quiz K (if needed)
+
+If the same team took third place in Quiz J and K, then Quiz L will be a two-team quiz to determine first and second place.
+
+Quiz M:        Third Quiz L, First Quiz L, Second Quiz L (if needed)
 
 ## Championship Quizzes
+A team must win twice to become the champion team.  All three teams will continue to quiz until one team wins twice.
```
Comment (scottpeterson) [inline content/quiz_meets.md:129]: I consolidated this A LOT.  This top line makes a lot of verbiage unnecessary: e.g. "if no teams wins the first two quizzes, everyone advances to a 3rd quiz"....I mean DUH.

File: content/quiz_meets.md [hunk 5]
```diff
@@ -25,5 +32,111 @@
...
+- If the same team wins the first two championship quizzes,
+    - 2nd place is determined by the most second places. If that is a tie, second place will be determined by Clarification (below).
```
Comment (scottpeterson) [inline content/quiz_meets.md:131]: I significantly tightened the 2nd place language. It makes it SUPER CLEAR that the determination of 2nd/3rd place when there's 3 quizzes is inconsistent.

File: content/quiz_meets.md (hunk 1 above)
Comment (gryphonshafer) [reply content/quiz_meets.md:10]: I'm thinking this is duplicated in the English-Script somewhere. Should this be part of scoring?

File: content/quiz_meets.md (hunk 2 above)
Comment (gryphonshafer) [reply content/quiz_meets.md:6]: WFM. But I'm thinking this needs to be English-Script and in the scoring section?

File: content/quiz_meets.md (hunk 1 above)
Comment (scottpeterson) [reply content/quiz_meets.md:10]: It's actually duplicated in this doc. Under Team Points Calculation.

If I remove the content under Preliminary Rounds, that's all that needs to happen right?

Comment (gryphonshafer) [reply content/quiz_meets.md]: You could remove the non-ES portion, yes. As long as you think the overall doc flow is still good.

Comment (scottpeterson) [reply content/quiz_meets.md]: I did, but kept a non-ES partial description.

**Pull request merged**